bioblend/_tests/TestGalaxyFolders.py

```diff
@@ -26,6 +26,12 @@ def test_show_folder(self):
         f2 = self.gi.folders.show_folder(self.folder['id'])
         self.assertEqual(f2['id'], self.folder['id'])
 
+    def test_show_folder_contents(self):
```

Comment:
wow, this is so new. I was looking at the code and the surrounding bits were very old